subseas_flex_fcst/layout.py

```diff
@@ -253,7 +253,7 @@ def navbar_layout(phys_units):
                     help_layout(
                         "Start Date",
                         "start_date",
-                        "Start date choices",
+                        "Model start dates",
```

Comment:
increase the Start Date box width so the date can fit in full.

Reply:
For these issues I was trying to figure out a way to make the size of the text or the size of the div change to accommodate changing the window size, so that they would always show the  full dates. Not so much luck on that though.

Reply:
That would be nice but maybe not worth sweating too much over it. What could be nice is to have some clean code to print such dates range in a more concise manner when can be. e.g.
8 Apr 2022 - 14 Apr 2022
could be
8-14 Apr 2022.
Whereas 31 Dec 2022 - 6 Jan 2023 can only keep that format...
That would mean checking if the years are equal, then if the month are equals, and append the date string accordingly.

As for saving space, I can think of 2 ways:
1/ One could be to apply slightly smaller font size.
2/ Another could be to create some menu widget like uicore does, where the title of the control is put on top of it
http://iridl.ldeo.columbia.edu/maproom/Global/ForecastsS2S/precip_median_subx.html?S=0000%2023%20Sep%202022
Or a combination of both.

I would not make those a priority unless it appears to be trivial.

Reply:
Okay I just pushed a commit that prints the date ranges more concisely. I am not seeing how putting the title of the control over the top will impact the display of the dates within the dropdown; if this is an aesthetic change you might like to have done still I can make it into an issue to address later.

Reply:
Okay should I make an issue for this then or is it not much worth thinking about in general?

Reply:
I think we can let go. I am sure there will be plenty of opportunities to remind us we can do much better aesthetically.... :)

Reply:
Okay. Then I think this PR is ready for approval unless any other things are found by you are Aaron to be changed.